jenkins/poc/haim/appeng/PolicyJenkinsfileDaily

```diff
@@ -35,12 +39,32 @@ pipeline {
         }
         stage('Run Policies the Cost Policies') {
             steps {
-                 sh 'python3 jenkins/poc/haim/common/run_cost_policies.py'
+                 script {
+                    def account = ['appeng']
+                    for (int i = 0; i < account.size(); ++i) {
+                        echo "Testing the ACCESS_KEY_${account[i]} browser"
+                        env.access_key = "ACCESS_KEY_${account[i]}"
+                        env.secret_key = "SECRET_KEY_${account[i]}"
+                        env.s3_bucket = "S3_BUCKET_${account[i]}"
```

Comment:
convert ${account} to uppercase by ${account[i].toUpperCase()}. It will match the env variable exported above.

Reply:
No longer needed